api/types/usertasks/descriptions/ec2-ssm-script-failure.md

```diff
@@ -0,0 +1,3 @@
+Teleport was able to reach the SSM Agent inside the EC2 instance, however the script returned an error.
```

Comment:
Also, how can a user see and update the script?
```suggestion
Teleport was able to reach the SSM Agent inside the EC2 instance, however the install script returned an error.
```

Reply:
I'll the InstallerScript used to the list of fields.
UI can suggest the user to do a `tctl edit installer/<script-name>` so that the user can edit the script.
What do you think?

I'm not so sure about this one as I am for the SSM Document, because for this one, the user needs to change tools (we don't expose CRUD for installer scripts in the UI).

I'll add them anyway.